packages/jobs/lib/execution/operations/handler.ts

```diff
@@ -0,0 +1,60 @@
+import type { JsonValue } from 'type-fest';
```

Comment:
mostly renaming the output.ts file into handler.ts (and removing the set task success/failed logic